src/components/Button/button.tsx

```diff
@@ -65,10 +27,14 @@ const CustomButton: FC<CustomButtonProps> = ({
                 hover={hover}
                 opacity={opacity}
                 width={width}
+                onClick={onClick}
+                fontSize={fontSize}
+                fontWeight={fontWeight}
+                textTransform={textTransform}
```

Comment:
Instead of passing styling props one-by-one use the `sx` object

Reply:
Done